### PR TITLE
[WIP] Add reload

### PR DESCRIPTION
**Pull request by Etheriq**

### CODE REVIEW

File: bin/reload.sh [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+#php app/console doctrine:database:drop --force
+
+#php app/console doctrine:database:create
+#php app/console doctrine:generate:entities Geekhub
+#php app/console doctrine:schema:update --force
+#php app/console cache:clear
+#php app/console doctrine:fixtures:load --no-interaction
+#php app/console assets:install --symlink
+#php app/console assetic:dump
+#php app/console cache:clear
```
Comment (spolischook) [inline bin/reload.sh]: Don't forget for empty line at the end.

**Comment by monobook** [comment]
what's about first line in reload.sh script?

#!/usr/bin/env php

**Comment by Etheriq** [comment]
Если можно пример *nix OS, где #!/bin/sh не сработает?

хотя согласен,  лучше исправить на #!/bin/bash

**Comment by monobook** [comment]
my comment was for line 10 but not for whole changeset

@Etheriq, could you please write you comment in English

**Pull request closed without merging**